src/Pages/Main/Components/Best/Best.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import '../mainComp.scss';
+
+class Best extends Component {
+  finalPrice = (
+    Math.round(
+      (this.props.price * (100 - this.props.howMuchSale) * 0.01) / 1000
+    ) * 1000
+  ).toLocaleString();
+
+  render() {
+    const {
+      name,
+      imgSrc,
+      isBest,
+      isNew,
+      isSale,
+      howMuchSale,
+      price,
+    } = this.props;
+    return (
+      <div className="Best">
+        <div
+          className="bestImage"
+          style={{
+            backgroundImage: `url(${imgSrc})`,
+          }}
+        >
+          <div className="labelContainer">
+            {isBest && <span className="isBest"> BEST </span>}
+            {isNew && <span className="isNew"> NEW </span>}
+            {isSale && <span className="isSale"> SALE </span>}
+          </div>
+          <div className="hoverContainer">
+            <div className="hoverHeart">
+              <i class="far fa-heart" />
+            </div>
+            <div className="hoverCart">
+              <i class="fas fa-shopping-cart" />
+            </div>
+          </div>
+        </div>
+        {howMuchSale > 0 && (
```

Comment:
0이 아닌 값일 경우 boolean값은 true 이죠? ㅎㅎ 
boolean값을 이용해 조건을 간결하게 작성해볼까요?
```suggestion
        {howMuchSale && (
```

Reply:
Boolean(howMuchSale)로 수정하였습니다₩~!!!!